example/src/main/java/org/apache/rocketmq/example/benchmark/TransactionProducer.java

```diff
@@ -153,35 +150,37 @@ private static Message buildMessage(final int messageSize) throws UnsupportedEnc
     }
 }
 
-class TransactionExecuterBImpl implements LocalTransactionExecuter {
-
-    private boolean ischeck;
-
-    public TransactionExecuterBImpl(boolean ischeck) {
-        this.ischeck = ischeck;
-    }
-
-    @Override
-    public LocalTransactionState executeLocalTransactionBranch(final Message msg, final Object arg) {
-        if (ischeck) {
-            return LocalTransactionState.UNKNOW;
-        }
-        return LocalTransactionState.COMMIT_MESSAGE;
-    }
-}
-
-class TransactionCheckListenerBImpl implements TransactionCheckListener {
+//class TransactionExecuterBImpl implements LocalTransactionExecuter {
```

Comment:
？

Reply:
The extra commented code has been removed in the latest commit，thanks for your reminding.